packaging/tags.py

```diff
@@ -225,10 +225,43 @@ def cpython_tags(
                 yield Tag(interpreter, "abi3", platform_)
 
 
-def _generic_abi() -> Iterator[str]:
-    abi = sysconfig.get_config_var("SOABI")
-    if abi:
-        yield _normalize_string(abi)
+def _generic_abi() -> List[str]:
```

Comment:
This is purposefully an iterator so the API is internally consistent for conceptual reasons (as well as not messing up when folks benefit from a lazy iteration). As such, please convert it back to an iterator.

Reply:
I was patterning this after `_cpython_abis(...) -> List[str]`. Should that also become an iterator? If not the code on line 249 below will have to do gymnastics to turn the list returned by `_cpython_abis()` into an iterator...

Reply:
Fair enough. Then don't worry about it, and if it bugs me enough I will make it internally consistent.